plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -73,9 +76,29 @@ public void execute(IProgressMonitor monitor) throws InvocationTargetException,
     
     setProjectIdPreference(newProject);
 
+    addAppEngineLibrariesToBuildPath(newProject, config.getAppEngineLibraries(), monitor);
+
     addJunit4ToClasspath(monitor, newProject);
   }
 
+  private void addAppEngineLibrariesToBuildPath(IProject newProject, List<Library> libraries, IProgressMonitor monitor) throws CoreException {
+    SubMonitor subMonitor = SubMonitor.convert(monitor, libraries.size());
+    subMonitor.setTaskName("Adding AppEngine libraries");
+    IJavaProject javaProject = JavaCore.create(newProject);
+    IClasspathEntry[] rawClasspath = javaProject.getRawClasspath();
+    IClasspathEntry[] newRawClasspath = Arrays.copyOf(rawClasspath, rawClasspath.length + libraries.size());
+    for (int i = 0; i < libraries.size(); i++) {
```

Comment:
I think you can use an enhanced for loop over libraries here since you aren't using the loop variable for anything except retrieving an element